_docs_integrate/01-connector-tutorial.md

```diff
@@ -8,47 +8,173 @@ In this tutorial we go through the basic steps necessary to establish a Relation
+
+## Connector: Test your Requests Validity
+
+In order to make sure the Request and its items are valid you can validate it calling the `POST /api/v1/Requests/Outgoing/Validate` route. Send the Request that will be used later under `onNewRelationship` in the Relationship Template.
 
 ```json
 {
-    "maxNumberOfRelationships": 10,
-    "expiresAt": "2022-06-01T00:00:00.000Z",
     "content": {
-        "attributes": [
+        "items": [
             {
-                "name": "Thing.name",
-                "value": "Contoso"
+                "@type": "RequestItemGroup",
+                "mustBeAccepted": true,
+                "title": "Shared Attributes",
+                "items": [
+                    {
+                        "@type": "ShareAttributeRequestItem",
+                        "mustBeAccepted": true,
+                        "attribute": {
+                            "@type": "IdentityAttribute",
+                            "owner": "",
+                            "value": {
+                                "@type": "DisplayName",
+                                "value": "ConnectorV2 Demo"
+                            }
+                        },
+                        "sourceAttributeId": "<the id of the attribute created above>"
+                    }
+                ]
+            },
+            {
+                "@type": "RequestItemGroup",
+                "mustBeAccepted": true,
+                "title": "Requested Attributes",
+                "items": [
+                    {
+                        "@type": "ReadAttributeRequestItem",
+                        "mustBeAccepted": true,
+                        "query": {
+                            "@type": "IdentityAttributeQuery",
+                            "valueType": "Surname"
+                        }
+                    },
+                    {
+                        "@type": "ReadAttributeRequestItem",
+                        "mustBeAccepted": true,
+                        "query": {
+                            "@type": "IdentityAttributeQuery",
+                            "valueType": "GivenName"
+                        }
+                    },
+                    {
+                        "@type": "ReadAttributeRequestItem",
+                        "mustBeAccepted": false,
+                        "query": {
+                            "@type": "IdentityAttributeQuery",
+                            "valueType": "EMailAddress"
+                        }
+                    }
+                ]
             }
-        ],
-        "request": {
-            "required": [
+        ]
+    }
+}
+```
+
+{% include rapidoc api_route_regex="^post /api/v1/Requests/Outgoing/Validate$" %}
+
+## Connector: Create a Relationship Template
+
+Start by creating a so called Relationship Template on the Connector. You can do so by calling the `POST /api/v1/RelationshipTemplates/Own` route. Use the following JSON in the request body:
```

Comment:
Ich würde vielleicht den Value von "onNewRelationship" mit einem Platzhalter ersetzen und rein schreiben, dass man den Wert aus dem vorigen Schritt einfügen soll. Das hat mehrere Vorteile:
1. Es wird klar, dass es sich hier um dieselbe Payload handelt
2. Falls der Nutzer im letzten Schritt was eigenes einführt hat, wird er hiermit dazu gezwungen, es zu kopieren
3. Wir müssen nicht dran denken, die Payload an beiden Stellen anzupassen, falls wir in Zukunft mal was daran ändern.

Reply:
done